tests/behat/features/access.feature

```diff
@@ -55,7 +55,7 @@ Feature: Access permissions
     Given I go to "user/password"
     Then I fill in "Username or email address" with "test@example.com"
     And I press "Submit"
-    And I should see the success message "If the username or email address exists and is active, further instructions have been sent to your email address."
+    And I should see the success message "If test@example.com is a valid account, an email will be sent with instructions to reset your password."
```

Comment:
this test has been updated related to the recent release and issue fix in username_enumeration_prevention. [The issue that has been fixed](https://www.drupal.org/project/username_enumeration_prevention/issues/3112961).

Reply:
cool!